.github/workflows/main.yml

```diff
@@ -76,19 +78,25 @@ jobs:
           zlib1g-dev \
           strace \
           libssl-dev \
-          pkg-config
+          pkg-config \
+          crossbuild-essential-amd64
 
     - name: Install rust
       run: |
         export RUSTUP_HOME="$HOME/.rustup"
         curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
         source "$HOME/.cargo/env"
-        cargo --version
         echo "RUSTUP_HOME=$HOME/.rustup" >> $GITHUB_ENV
         echo "PATH=$HOME/.cargo/bin:$PATH" >> $GITHUB_ENV
+        cargo --version
 
     - name: Update postgrestd git submodules
       run: git submodule update --init --recursive
 
     - name: Build postgrestd
-      run: ./run install
+      # run: STD_TARGETS=aarch64-postgres-linux-gnu ./run install
```

Comment:
Let's get a comment on why we are doing it like this.

Reply:
Ok, Ill add some comments when I go to clean it up